doc/index.rst

```diff
@@ -1,17 +1,17 @@
-***********************************************
+********************************************
 Welcome to the NEST simulator documentation!
-***********************************************
+********************************************
 
 :orange:`Introducing NEST 3.0!`
 -------------------------------
 
 Our aim with NEST 3.0 is to provide a more intuitive experience with simplified yet versatile handling and manipulation of nodes and connections.
```

Comment:
Be more direct: "NEST 3.0 provide a more intuitive ..."

Reply:
Done.